pkg/pool-linear/contracts/aave/AaveLinearPoolFactory.sol

```diff
@@ -57,14 +70,31 @@ contract AaveLinearPoolFactory is
         _poolVersion = poolVersion;
     }
 
+    /**
+     * @dev Return the address of the most recently created pool.
+     */
     function getLastCreatedPool() external view override returns (address) {
         return _lastCreatedPool;
     }
 
+    /**
+     * @dev Return the pool version deployed by this factory.
+     */
     function getPoolVersion() public view override returns (string memory) {
         return _poolVersion;
     }
 
+    /**
+     * @dev Return the name associated with the given protocolId, if registered.
+     */
+    function getProtocolName(uint256 protocolId) external view returns (string memory) {
+        ProtocolIdData storage protocolIdData = _protocolIds[protocolId];
```

Comment:
why not `memory`?

Reply:
It could be memory, but we're not changing anything, so this should avoid a copy. (It already has to copy the string out.)

Reply:
It doesn't really matter, but I'd use `memory` here. We don't care about gas, and you typically see storage pointers when there's writes involved.